src/handlers/ecommerce/payment-method.ts

```diff
@@ -7,3 +8,63 @@ export const ecommerceGetPaymentMethods: RequestHandler = async (req, res) => {
   logger.info("[Get payment-methods ecommerce] - Return success case");
   res.status(200).send(createSuccessGetPaymentMethods());
 };
+
+const { X_API_KEY } = process.env;
+
+export const createFormWithNpg: RequestHandler = async (_req, res) => {
+  logger.info(
+    `[Get Payment Method preauthorization id: ${_req.params.id}] - Return success case`
+  );
+  const options = {
+    headers: {
+      "Content-Type": "application/json",
+      "Correlation-Id": "df615d11-1159-43c9-bd07-fdc6f460e449",
+      "X-Api-key": X_API_KEY
+    },
+    hostname: "stg-ta.nexigroup.com",
+    maxRedirects: 20,
+    method: "POST",
+    path: "/api/phoenix-0.0/psp/api/v1/orders/build"
+  };
+
+  const build = https.request(options, response => {
```

Comment:
Why not use `fetch`?

Reply:
I started from #52 solution that implement this kind of request.

Reply:
I would still use `fetch` as it is less error prone and more readable

Reply:
did it!